pkg/kubectl/cmd/version.go

```diff
@@ -37,6 +37,14 @@ type Version struct {
 	ServerVersion *apimachineryversion.Info `json:"serverVersion,omitempty" yaml:"serverVersion,omitempty"`
 }
 
+// VersionConfig: describe the options available to users of the "kubectl
+// version" command.
+type VersionConfig struct {
```

Comment:
I still prefer `VersionOptions`.
Please file a PR to update `kubectl-conventions.md` to avoid confusing other contributors in the future. Thanks.